f2/src/summary/HowDidItStartSummary.js

```diff
@@ -36,11 +36,11 @@ export const HowDidItStartSummary = (props) => {
     )
 
     overviewLine =
-      i18n._('confirmationPage.howDidItStart.overviewPrefix') +
+      <Trans id="confirmationPage.howDidItStart.overviewPrefix" /> +
```

Comment:
When constructing strings, I think we should keep using `i18n._()`. Otherwise we end up with this: 
![Capture d’écran, le 2020-05-27 à 14 37 11](https://user-images.githubusercontent.com/5230720/83059364-c8210f80-a027-11ea-8ee5-4e7837123e72.png)